dev/com.ibm.ws.concurrent.persistent_fat_multiple/fat/src/com/ibm/ws/concurrent/persistent/fat/multiple/NOOPJDBCContainer.java

```diff
@@ -0,0 +1,106 @@
+package com.ibm.ws.concurrent.persistent.fat.multiple;
+
+import java.util.concurrent.Future;
+
+import org.testcontainers.containers.JdbcDatabaseContainer;
+
+/**
+ * This is a NO-OP Database Test Container that is returned
+ * when attempting to create a database test container for
+ * an unsupported database or to test against derby embedded.
+ * 
+ * This test container overrides the start and stop methods
+ * to prevent the creation of a docker container. 
+ * 
+ * Returns connection information for a derby embedded driver. 
+ *  
+ * TODO: Since this class is general it should be moved to fattest simplicity in the future
+ */
+class NOOPJDBCContainer<SELF extends NOOPJDBCContainer<SELF>> extends JdbcDatabaseContainer<SELF> {
+	
+	private String databaseName = "test";
+
+	/**
+	 * @see NOOPJDBCContainer
+	 */
+	public NOOPJDBCContainer(final Future<String> image) {
+		super(image);
+	}
+	
+	/**
+	 * @see NOOPJDBCContainer
+	 */
+	public NOOPJDBCContainer(String image) {
+		super(image);
+	}
+	
+	@Override
+	public String getDockerImageName() {
+		return "";
+	}
+	
+	@Override
+	public void start() {
+		//DO NOTHING
+	}
+	
+	@Override
+	protected void doStart() {
+		//DO NOTHING
+	}
+	
+	@Override
+	public void stop() {
+		//DO NOTHING
+	}
+	
+	@Override
+	public void close() {
+		//DO NOTHING
+	}
+	
+	@Override
+	public SELF withDatabaseName(String name) {
+		this.databaseName = name;
+		return self();
+	}
+	@Override
+	public String getDatabaseName() {
+		return "memory:" + this.databaseName;
```

Comment:
lets not assume an in-memory database is desired. Instead, the caller can do `withDatabaseName("memory:foo")` if they want